### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1]  Dialog: Not provided with accessible name.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11280]
#### Is this a bug, enhancement, or feature request?
Bug
#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/core/dialog
Open any dialog
Below code found by Access assistant tool
This DIV (role=dialog) does not have a mechanism that allows an accessible name value to be calculated.

Code Snippet: <div style="overflow: auto; position: relative; width: 680px; height: auto;" class="cdk-drag fd-scrollbar fd-dialog__content cdk-drag-disabled fd-dialog__content--m" fd-scrollbar="" cdkdragboundary=".fd-dialog" cdkdrag="" cdktrapfocus="" fdkresize="" aria-modal="false" tabindex="0" role="dialog"><div class="cdk-drag-handle" cdkdraghandle=""><div _ngcontent-ng-c3990766896=""><div class="fd-bar fd-bar--header fd-dialog__header is-compact" fd-bar=""><div class="fd-bar__left ng-star-inserted" fd-bar-left=""><div class="fd-bar__element--full-width fd-bar__element"><h2 class="fd-title fd-title--h3" fd-title="" _ngcontent-ng-c3990766896="">Schedule Job</h2></div></div><div class="fd-bar__right ng-star-inserted" fd-bar-right=""><div class="fd-bar__element"></div></div></div></div></div><div style="overflow: auto; padding: 1rem;" class="fd-scrollbar fd-dialog__body" _ngcontent-ng-c3990766896=""><div _ngcontent-ng-c3990766896=""><div class="fd-busy-indicator__container" aria-live="polite" aria-busy="false" tabindex="-1" role="presentation" _ngcontent-ng-c3990766896=""><div class="fd-form-item" fd-form-item="" _ngcontent-ng-c3990766896=""><div _ngcontent-ng-c3990766896=""><div class="fd-form-item ng-star-inserted" fd-form-item="" _ngcontent-ng-c3990766896=""><label class="fd-form-label__wrapper" id="fd-form-label-31" for="startTime" fd-form-label="" _ngcontent-ng-c3990766896=""><span class="fd-form-label fd-form-label--required">Schedule:</span></label><div _ngcontent-ng-c3990766896=""><div id="fd-popover-36" class="fd-popover-custom fd-form-input-message-group is-cozy" additionalbodyclass="fd-popover__body--input-message-group"><div class="fd-popover__control ng-star-inserted" cdkoverlayorigin=""><div><div class="ng-pristine ng-valid ng-touched" placeholder="Provide schedule date" inputid="dateTimePicker" _ngcontent-ng-c3990766896=""><div class="fd-datetime"><div id="fd-popover-37" class="fd-popover-custom is-cozy"><div class="fd-popover__control ng-star-inserted" cdkoverlayorigin=""><div><div class="is-cozy" glyph="date-time"><div class="fd-input-group fd-input-group--control"><input aria-la.

Developers must either change the containing web page's title element (when the dialog lacks a visual title) to a terse description of the dialog purpose, or use an explicit heading element as the dialog title (preferably h1 or h2) (when a visual dialog title is present) to provide a title for the dialog. In addition, any heading should be referenced via aria-labelledby from the element representing the dialog, e.g. element with role of dialog or role of region. In many cases the dialog's title does not already appear as an explicit heading, and needs to be tagged with a heading element to meet other accessibility best practices.

When using ARIA role dialog or role of region, developers must provide a dialog label using aria-label or aria-labelledby attributes. The dialog's aria-labelledby should not reference the entire dialog itself as this can cause double speaking by some screen readers. The accessible name of the dialog should be the dialog title and not the dialog's contents.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
•	Download Access assistant tool from level access.
•	For registered AMP user, login with your SAP email ID into SAP AMP instance: sap.levelaccess.net
Open any dialog:
This DIV (role=dialog) does not have a mechanism that allows an accessible name value to be calculated

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

I checked the first example on the page.
The dialog has aria-labelledby attribute and is labelled, which is good.
Still, for some reason the paragraph inside has also a role dialog, which is wrong and needs to be corrected. And I assume that this causes the issue here, as this "dialog" which does not have to be a dialog is unlabelled.
![image](https://github.com/SAP/fundamental-ngx/assets/17043004/318e9212-d0d9-4414-8f92-a0387902eb85)

Best Regards,
Nikolay
